websites/Z/ZappingTV/presence.ts

```diff
@@ -0,0 +1,85 @@
+const presence = new Presence({
+		clientId: "1051342250870853702",
+	}),
+	elapsed = Math.floor(Date.now() / 1000),
+	getElement = (query: string): HTMLElement => {
+		return document.querySelector(query);
+	},
+	getEndTime = (): number => {
+		const [h, m] = getElement(".control-end-time")
+			.textContent.split(":")
+			.map(x => parseInt(x));
+		return Math.floor(new Date().setHours(h, m, 0, 0) / 1000);
+	},
+	getChannelData = async ({
+		coloredLogos = false,
+	}): Promise<{
+		logo: string;
+		title: string;
+		channel: string;
+	}> => {
+		const logo: HTMLImageElement = document.querySelector(
+			"li.btn-channel.active > div.channel-image > img"
+		);
+		return {
+			logo: coloredLogos
+				? logo.src.replace("white", "color").replace("png", "jpg")
+				: logo.src,
+			title: getElement(".control-title").textContent.trim(),
+			channel: getElement(
+				"li.btn-channel.active > div.channel-data > div.name"
+			).textContent.trim(),
+		};
+	},
+	assets: {
+		logo: string;
+		sleep: string;
+	} = {
+		logo: "https://i.imgur.com/5gyFfcd.png",
+		sleep: "https://i.imgur.com/gWBjy43.png",
+	};
```

Comment:
Prefer an `enum` over this:
https://github.com/PreMiD/Presences/blob/63c0d808d092ca9e4a40feadd8caba231c4d2a5c/websites/C/Crunchyroll/presence.ts#L52-L58

Reply:
https://github.com/sebalvaro/Presences/blob/8bb7ebba3a0773052695650f4f08324ce64da0ce/websites/Z/ZappingTV/presence.ts#L34-L37

Reply:
Done